perun-openapi/openapi.yml

```
@@ -8491,6 +8559,29 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/extSourcesManager/removeExtSources/g-s:
+    post:
+      tags:
+        - ExtSourcesManager
+      operationId: removeExtSourcesWithGroupSource
+      summary: Remove associations of external source definitions with a GROUP.
+      parameters:
+        - $ref: '#/components/parameters/groupId'
+        - name: sourceIds
+          description: "list of ext source ids"
+          schema:
+            type: array
+            items:
+              $ref: '#/components/parameters/sourceId'
+          in: query
+          required: true
```

You can shorten all these four methods for extsources by defining a list of ext sources between parameters, something like that:
```
    extSourceIds:
      name: "extSources[]"
      description: "list of extSource ids List<Integer>"
      in: query
      required: true
      schema:
        type: array
        items:
          type: integer
```
```suggestion
        - $ref: '#/components/parameters/extSourceIds'
```